app/src/main/kotlin/com/hedvig/app/feature/checkout/CheckoutViewModel.kt

```diff
@@ -180,8 +168,6 @@ class CheckoutViewModel(
   }
 
   private suspend fun onSignSuccess(): Event.CheckoutSuccess {
-    featureManager.invalidateExperiments()
-    loginStatusService.isLoggedIn = true
```

Comment:
sign success no longer means we're logged in. But I also thought it shouldn't have something to do with invalidating the feature flags, so deleted.